src/utils/barcode.ts

```diff
@@ -0,0 +1,41 @@
+import { createCanvas } from "canvas";
+import * as t from "io-ts";
+import * as JsBarcode from "jsbarcode";
+import { DOMImplementation, XMLSerializer } from "xmldom";
+
+export const BarcodeOutput = t.interface({
+  png: t.string,
+  pngMimeType: t.string,
+  svg: t.string,
+  svgMimeType: t.string
+});
+
+export type BarcodeOutput = t.TypeOf<typeof BarcodeOutput>;
+
+export function toBarcode(text: string): BarcodeOutput {
```

Comment:
can this method throw exceptions ? can you make it return an Either in case ? (tryCatch2v)